### PR TITLE
move examples to WaterLily-Examples

### PR DESCRIPTION
**Pull request by marinlauber**

### CODE REVIEW

**Comment by weymouth** [comment]
This is showing up as breaking the CI. Is that just the code coverage?

**Comment by marinlauber** [comment]
Yes, my bad, the documenter was using some stuff in `examples/` it's sorted now.

**Comment by weymouth** [comment]
Do we want to clean up the README since all of those examples are now in another repo? Maybe we can focus the WaterLily README and point them straight over to the tutorials?

File: README.md [hunk 1]
```diff
@@ -29,232 +29,110 @@ If you have used WaterLily for research, please __cite us__! The following manus
...
+    body = AutoBody((x,t)->√sum(abs2, x .- center) - radius)
+    Simulation((n,m),(U,0),2radius;ν=U*2radius/Re,body,
+                mem, # memory type
+                T)   # Floating point type
+end
 
-# don't forget to close the file
-close(writer)
+using CUDA
+@assert CUDA.functional()   # is your CUDA GPU working??
```
Comment (marinlauber) [inline README.md:124]: Is that still useful? The extension does it when you load it already, right?

Comment (weymouth) [reply README.md:124]: On my machine at home, `using CUDA` doesn't throw any error, so the `@assert` is needed to stop people from following the example blindly.

**Pull request merged**